cmakemodules/script_install_commands.cmake

```diff
@@ -45,6 +45,13 @@ IF(WIN32)
 		ENDFOREACH(F)
 	ENDIF (EXISTS "${wxWidgets_LIB_DIR}")
 
+	# Qt5:
+	IF (${CMAKE_MRPT_HAS_Qt5})
+		FOREACH(F ${Qt5_COMPONENTS_LIBS})
```

Comment:
Will this cause a problem with Linux?
I think you don't want to do the install of the components on Linux.